### ISSUE TITLE
Interface Spelling Error: "TackMeNot"

### ISSUE THREAD

**Issue by dhowe** [vtoubiana/TrackMeNot#13]
FILE: data/tmn_widget.html

```
<img src="images/skin/tmn_icon.png" id="tmn-widget-icon">
<label id="tmn-text" style="vertical-align:top" >TackMeNot</label>
</div>
</body>
</html>
```

**Comment by dunn** [comment]
Fixed in e91a87b838969a36146ff0cfe3d42e734b9d5537?

**Comment by vtoubiana** [comment]
Fixed (the label never appears anyway)
